src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -8,142 +8,216 @@ folder: setup-docker
+* The initialization phase is executed by the `init` command. It installs version-specific files into the directory mounted to `/data`. This includes the universal configuration file named `che.env`, a version identifier, and a location where configuration files are saved. The configuration is executed by the `config` command. It takes as input, the `che.env` configuration file, the OS of your host, and then generates an OS-specific set of configuration files in the `/data/instance` directory that can be used to run an instance of Che. 
 
-This command respects `--no-force`, `--pull`, `--force`, and `--offline`.
+* The configuration phase runs an initialization if a directory is not found. Every execution of the `config` command overwrites the files in the `/data/instance` directory with the latest configuration. This ensures that if an administrator modifies any configuration file, the instance’s configuration files are updated consistently. The CLI generates a large number of configuration files specific to running Che. The configuration files are sourced from the puppet templates that are stored in the GitHub repository under `/dockerfiles/init`. 
 
-'''''
+* The start phase is executed by the `start` command and it uses a configuration-generated `docker-compose-container.yml` file to launch Eclipse Che. The start phase always executes a `config` command, so any files that were edited in the `/data/instance` directory are overwritten with the generated configuration from the CLI.
 
-_destroy_
+The CLI hides most error conditions from standard out. Internal stack traces and error output are redirected to the `cli.log` file, which is saved in the host directory where `:/data` is mounted.
 
-Deletes `/docs`, `che.env` and `/instance`, including destroying all user workspaces, projects, data, and user database. If you pass `--quiet` then the confirmation warning will be skipped. Passing `--cli` will also destroy the `cli.log`. By default this is left behind for traceability.
+You can override any value in the `che.env` file for a single execution by passing the `-e NAME=VALUE` argument on the command line. The CLI detects the values on the command line and ignores those imported from the `che.env` file.
 
-'''''
+Following are some selected commands and their uses.
 
-_dir_
+*_action_*
 
-Boots a new Eclipse Che instance with a workspace for the folder `:/chedir` defined as volume mount in parameter.
+The `action` command executes some actions on the Eclipse Che instance or on a workspace running inside Che. To list all workspaces on Che, use the `action list-workspaces` command. To execute a command on a workspace, use the  `action execute-command <workspace-name> <action>` command; here, action is a `bash` command.
```

Comment:
In a shell, `bash` itself could be a command -- do you mean that `<action>` is a command that will be interpreted by Bash?